tests/App/AppKernel.php

```diff
@@ -33,6 +33,11 @@
 use Symfony\Component\Routing\RouteCollectionBuilder;
 use Symfony\Component\Security\Http\Authentication\AuthenticatorManager;
 
+/**
+ * @psalm-suppress PropertyNotSetInConstructor
+ *
+ * @see https://github.com/psalm/psalm-plugin-symfony/pull/220
```

Comment:
I've created https://github.com/psalm/psalm-plugin-symfony/issues/225

Reply:
Let's see if it gets solved, the PR I linked it has the problem described as the last message too.